README.md

```diff
@@ -1,18 +1,28 @@
 # Evals
 
-Evals is a framework for evaluating OpenAI models and an open-source registry of benchmarks.
+Evals is a framework for evaluating large language models (LLMs) and LLM Programs. It also includes an open-source registry of challenging benchmark evals.
```

Comment:
We should define what `LLM Programs` are, or get rid of it since we don't use this terminology elsewhere. It's a bit confusing since CompletionFn is essentially LLM program AFAICT.

Reply:
How about "Evals is a framework for evaluating large language models (LLMs) and other prompt completion programs"

Reply:
I decided to go with `Evals is a framework for evaluating large language models (LLMs) and other prompt completion functions`